pkgs/by-name/sp/spice-autorandr/package.nix

```
@@ -46,6 +46,6 @@ stdenv.mkDerivation  {
     maintainers = with lib.maintainers; [
       dmytrokyrychuk
     ];
-    platforms = [ "x86_64-linux" ];
+    platforms = [ "aarch64-linux" "x86_64-linux" ];
```

```suggestion
    platforms = lib.platforms.linux;
```

It should work on all linux platforms right?

---

I guess? As far as I know there is no explicit statement from upstream in this regard.

---

It just happened that I needed it and I could check it built just fine. 

I think it’s reasonable to assume it should work on all Linux platforms.